### ISSUE TITLE
Is there a pre-trained model trained on coQA available

### ISSUE THREAD

**Issue by abhishek-jha13** [stevezheng23/xlnet_extension_tf#79]
I want to use pre-trained question answering model trained on coQA like hugging face. Do we have something available?

**Comment by stevezheng23** [comment]
Sorry, I haven't released trained CoQA model to public yet

**Comment by abhishek-jha13** [comment]
No Worries, I was able to train the model  using your code

**Comment by abhishek-jha13** [comment]
No Worries, I was able to train the model  using your code

**Comment by stevezheng23** [comment]
Great!

**Comment by imagine3D-ai** [comment]
> No Worries, I was able to train the model using your code

what accuracy were you able to achieve?

**Comment by imagine3D-ai** [comment]
> Sorry, I haven't released trained CoQA model to public yet

Hi! I’m wondering if you’vr released trained CoQa model by now?

**Comment by abhishek-jha13** [comment]
> > No Worries, I was able to train the model using your code
> 
> what accuracy were you able to achieve?

I was able to achieve an F1 score of 87. If you need my help, email me at abhishekjha13@gmail.com

**Comment by imagine3D-ai** [comment]
> Sorry, I haven't released trained CoQA model to public yet

I'm trying to train XLNET on COQA using your 'run_coqa' script, but my Google Colab notebooks keep crashing. I tried using both CPU and GPU notebook instances and changing batch_size to 1, but they still keep crashing and showing the:

'Allocation of 16777216 exceeds 10% of system memory.' error.

Did you face any similar memory usage errors and did you manage to fix them?

**Comment by abhishek-jha13** [comment]
There is bug in the code which causes an infinite loop which causes crashing. Try to fix that.

**Comment by imagine3D-ai** [comment]
Hmm, do you mean in my code or in the xlnet library?

**Comment by abhishek-jha13** [comment]
There is a bug in the run_coqa.py

**Comment by imagine3D-ai** [comment]
I’m really struggling to find it. Could you make a pull-request on this
repo, so that this can be patched?

**Comment by stevezheng23** [comment]
> There is a bug in the run_coqa.py

Please feel free to send out a PR for fixing the bug, thanks!
